plugins/com.google.cloud.tools.eclipse.appengine.facets/src/com/google/cloud/tools/eclipse/appengine/facets/ui/AppEngineStandardProjectConvertCommandHandler.java

```diff
@@ -38,26 +40,33 @@
 
   @Override
   public Object execute(ExecutionEvent event) throws ExecutionException {
+    Shell shell = HandlerUtil.getActiveShellChecked(event);
     try {
       IProject project = ProjectFromSelectionHelper.getProject(event);
       if (project == null) {
         throw new NullPointerException("Convert menu enabled for non-project resources");
       }
 
+      // check if Cloud SDK is available, prompts to configure if not
```

Comment:
This comment is not 100% correct in that it just checks if a user has ever set any path to the Cloud SDK preference. That is, if a user set an invalid path, this check passes even if the Cloud SDK is not available. What do you think?